content/home/2-what-is-nuts.md

```diff
@@ -2,17 +2,12 @@
 title: Wat doet Nuts?
 ---
 
-Nuts ontwikkelt en beheert een nutsvoorziening die het delen van
-zorg-gerelateerde informatie over het Internet mogelijk maakt op een
-vertrouwelijke, veilige en toegankelijke manier. Met als doel om digitale
-samenwerking in de gehele zorg te faciliteren — en zo de zorg te verbeteren.
+Nuts is een open [community](./community) die wordt ondersteund door een [stichting](./stichting).
```

Comment:
We wilden af van "Nuts is...". Iets over een zelfstandig naamwoord ofzo dat ik zelf ook niet snap. Maar we kregen het niet voor elkaar om in één of twee zinnen te zeggen wat Nuts "is". Daarom heb ik er uiteindelijk van gemaakt wat Nuts "doet".